lib/scope_file_access_archive.rb

```diff
@@ -32,6 +32,16 @@ def export_files_as_tarballs(destination)
     end
   end
 
+  def has_file?(name)
+    found = files.any? { |file| file.name == name }
+    return true if found
+    if files.any? { |file| file.name == File.join(File.dirname(name), '') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.